src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/__tests__/ExposureSubmissionTestResultViewControllerTests.swift

```diff
@@ -22,11 +22,18 @@ import XCTest
 @testable import ENA
 
 class ExposureSubmissionViewControllerTests: XCTestCase {
+	
```

Comment:
Use`MockTestStore` and add linebreaks after parameters in this file as well

Reply:
Done